doc/WinGetConfigure.md

```diff
@@ -0,0 +1,137 @@
+
+### Ensure you are using PowerShell 7.2 or later
+
+First you need to have PowerShell 7.2 or later. You can run `$PSVersionTable` in PowerShell if you see the following result, you need to switch to a more recent version of PowerShell or install one.
+
+```PowerShell
+PS C:\Users\denelon> $PSVersionTable
+
+Name                           Value
+----                           -----
+PSVersion                      5.1.22621.963
+PSEdition                      Desktop
+PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
+BuildVersion                   10.0.22621.963
+CLRVersion                     4.0.30319.42000
+WSManStackVersion              3.0
+PSRemotingProtocolVersion      2.3
+SerializationVersion           1.1.0.1
+```
+
+To install or upgrade PowerShell to the latest version run `winget install Microsoft.PowerShell`.
+```PowerShell
+Found PowerShell [Microsoft.PowerShell] Version 7.3.3.0
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://github.com/PowerShell/PowerShell/releases/download/v7.3.3/PowerShell-7.3.3-win-x64.msi
+  ██████████████████████████████   101 MB /  101 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+*Note: At the time of this writing, the latest version available is 7.3.3.0
+
+Launch PowerShell and run `$PSVersionTable` to verify PSVersion is greater than 7.2:
+
+```PowerShell
+PS C:\Users\denelon> $PSVersionTable
+
+Name                           Value
+----                           -----
+PSVersion                      7.3.3
+PSEdition                      Core
+GitCommitId                    7.3.3
+OS                             Microsoft Windows 10.0.22621
+Platform                       Win32NT
+PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
+PSRemotingProtocolVersion      2.3
+SerializationVersion           1.1.0.1
+WSManStackVersion              3.0
+```
+
+### Install the PSDesiredStateConfiguration module
+
+These instructions are taken from [Get started with invoking DSC resources](
+https://learn.microsoft.com/powershell/dsc/getting-started/invoking-dsc-resources?view=dsc-2.0).
+
+Now you will need to install PSDesiredStateConfiguration by running
+`Install-Module -Name PSDesiredStateConfiguration -RequiredVersion 2.0.6`
+
+If you see:
+```PowerShell
+Untrusted repository
+You are installing the modules from an untrusted repository. If you trust this repository, change its
+ InstallationPolicy value by running the Set-PSRepository cmdlet. Are you sure you want to install
+the modules from 'PSGallery'?
+[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"):
+```
+
+In this case, you can respond with "Y" if you trust the PowerShell gallery to install this module.
```

Comment:
Can we link to the PowerShell gallery homepage or some docs around what it is in case users aren't familiar?

Reply:
Done